scripts/run_e2e_tests.py

```diff
@@ -219,7 +219,6 @@ def run_tests(args: argparse.Namespace) -> Tuple[List[bytes], int]:
             automatic_restart=False,
             skip_sdk_update_check=True,
             env={
-                **os.environ,
```

Comment:
This is now a discrepancy since the environment variable merging is no longer happening in servers.py.